{"name": "test_reg_russian_username", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Incorrect username' in '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...ize:10px\">powered by ТЕХНОАТОМ</p>\\n    <p>Readability counts</p>\\n</div>\\n            </footer>\\n    \\n</body></html>'\n +  where '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...ize:10px\">powered by ТЕХНОАТОМ</p>\\n    <p>Readability counts</p>\\n</div>\\n            </footer>\\n    \\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"0f6aa6141dc9212b692b04485ace6f60\")>.page_source\n +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"0f6aa6141dc9212b692b04485ace6f60\")> = <tests.test_ui.TestReg object at 0x10f55ec50>.driver", "trace": "self = <tests.test_ui.TestReg object at 0x10f55ec50>\n\n    def test_reg_russian_username(self):\n        \"\"\"\n        Тестируется возможность пройти регистрацию с русским значением поля username.\n        Приложение должно выдавать ошибку в силу следующих причин:\n        1. В любом сервисе данное поле заполняется на универсальном английском языке и почти всегда должно\n        быть уникально(это в данном случае предусмотрено)\n        2. Приложение подразумевает использование VK API, для работы которого необходимо передавать username на англ.\n        \"\"\"\n        username, email, password = generate_user(russian_username=True)\n        self.reg_page.reg(username, email, password)\n        self.builder.del_user(username)\n        allure.attach(self.driver.get_screenshot_as_png(),\n                      name='test_python_link',\n                      attachment_type=allure.attachment_type.PNG)\n>       assert \"Incorrect username\" in self.driver.page_source\nE       assert 'Incorrect username' in '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...ize:10px\">powered by ТЕХНОАТОМ</p>\\n    <p>Readability counts</p>\\n</div>\\n            </footer>\\n    \\n</body></html>'\nE        +  where '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...ize:10px\">powered by ТЕХНОАТОМ</p>\\n    <p>Readability counts</p>\\n</div>\\n            </footer>\\n    \\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"0f6aa6141dc9212b692b04485ace6f60\")>.page_source\nE        +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"0f6aa6141dc9212b692b04485ace6f60\")> = <tests.test_ui.TestReg object at 0x10f55ec50>.driver\n\ntests/test_ui.py:91: AssertionError"}, "description": "\n        Тестируется возможность пройти регистрацию с русским значением поля username.\n        Приложение должно выдавать ошибку в силу следующих причин:\n        1. В любом сервисе данное поле заполняется на универсальном английском языке и почти всегда должно\n        быть уникально(это в данном случае предусмотрено)\n        2. Приложение подразумевает использование VK API, для работы которого необходимо передавать username на англ.\n        ", "attachments": [{"name": "test_python_link", "source": "bede2954-9ffc-4d37-a118-9e0d7f032b8f-attachment.png", "type": "image/png"}], "start": 1590601675665, "stop": 1590601694898, "uuid": "421e178e-7e91-4f3b-b6e2-0b71d47538e0", "historyId": "a80deb18a3fa2089d070cc670f7385a4", "testCaseId": "040c3145c8c4500e0bed14c3faa5a5f8", "fullName": "tests.test_ui.TestReg#test_reg_russian_username", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestReg"}, {"name": "host", "value": "MacBook-Air-Roman-2.local"}, {"name": "thread", "value": "20109-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}